resourcepolicies.md

```diff
@@ -10,34 +10,298 @@ As we don't have yet an use case to iterate over all the existent resource polic
   "policyType": "TYPE_SUBMISSION",
   "action": "READ",
-  "type": "resourcePolicy"
+  "startDate" : null,
+  "endDate" : null,
+  "type": "resourcepolicy"
 }
 ```
 
+Attributes
+* name: a short name associated to describe the policy
+* description: a description for the policy, i.e. the reason for the policy
+* policyType: a classification of the policy meanly from the process that has set it. One of 
+** TYPE_SUBMISSION: a policy in place during the submission 
+** TYPE_WORKFLOW: a policy in place during the approval workflow
+** TYPE_INHERITED: a policy that has been inherited from a container (the collection)
+** TYPE_CUSTOM: a policy defined by the user during the submission or workflow phase
+* action: the action grant by the policy. One of READ, WRITE, ADD, REMOVE, ADMIN, DEFAULT_BITSTREAM_READ, DEFAULT_ITEM_READ as defined in the Constants class
+* startDate: the first day of validity of the policy, if null the policy is assumed valid from ever. The date will be in the format YYYY-MM-DD
+* endDate: the last day of validity of the policy, if null the policy is assumed valid for ever. The date will be in the format YYYY-MM-DD
+
 Exposed links:
 * eperson: link to the eperson that have permission grant by the policy
 * group: link to the group that have permission grant by the policy
 * resource: link to the resource subject to the policy
 
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators, users with ADMIN right on the target resource, users mentioned in the policy (eperson or member of the group) can access the resourcepolicy
+* 404 Not found - if the resourcepolicy doesn't exist (or was already deleted)
+
+### Search methods
+#### resource
+**/api/authz/resourcepolicies/search/resource?uuid=<:uuid>[&action=<:action>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the resource object of the policy
+* action: optional, limit the returned policies to the specified action (see GET description for allowed values)
+
+It returns the list of matching resource policies
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators and users with ADMIN right on the target resource can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### eperson
+**/api/authz/resourcepolicies/search/eperson?uuid=<:uuid>[&resource=<:resource>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the eperson that benefit of the policy
+* resource: optional, limit the returned policies to the ones related to the specified resource
+
+It returns the list of matching resource policies
```

Comment:
I wonder if we should add a note here to clarify that this only returns *explicit* policies (and not implicit or inherited policies) for this EPerson on this Resource.  So, for example, if the EPerson is inheriting permissions on a Resource from a *Group*, then this endpoint may return an empty list, while a Group-based search would return the actual policy.